pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -83,8 +83,15 @@ func parseControllerCerts(ctx *zedagentContext, contents []byte) (changed bool,
 				break
 			}
 		}
-		if !found {
-			log.Functionf("parseControllerCerts: deleting %s", config.Key())
+		// XXX move this inside if statement
+		// Is the ControllerCert in use?
+		if lookupCipherContextByCCH(ctx.getconfigCtx, configHash) != nil {
+			log.Noticef("XXX ControllerCert %s in use",
+				string(configHash))
+			found = true
```

Comment:
it looks like setting `found` here doesn't have any effect on the logic

Reply:
Agreed. Plan per the comment is to move this around but for now I wanted to have the logs to watch it in operation. FWIW this needs to be tested with a cert roll on some local cluster to make sure it does the right thing.

Reply:
@europaul cleaned this up. Please look again.